.agent/workflows/load-skills-overview.md

```
@@ -2,7 +2,7 @@
 description: Load portable overview skills and framework-scoped overview skills declared by the profile
 ---
 
-> **Estimated context: ~8.7K tokens with profile-declared DRN overview skills** (~4.5K portable-only)
+> **Estimated context: ~11.8K tokens with profile-declared DRN overview skills** (~7.2K portable-only)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)